AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/CallActivity.java

```diff
@@ -197,6 +197,8 @@ private void setStatusBarVisibility() {
     @Override
     public void onPause() {
         Log.d(LOG_TAG, "CallActivity onPause");
+        callingContext.pauseVideo().thenRun(() -> runOnUiThread(() ->
```

Comment:
For the scenario where both `onPause()` and `onStop()` is both called, `callingContext.pauseVideo()` will be called 2x.
Is there a way to detect when `callingContext.pauseVideo()` has already been called ?